babel.config.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env node */
```

Comment:
@rwwagner90 don't want to be show stopper, from https://github.com/babel/ember-cli-babel/issues/418 I got impression that addons should not ship `babel.config.js`